src/engine/UI/DisplayLayout.cpp

```diff
@@ -13,8 +13,9 @@ void DisplayLayout::AddElement(std::shared_ptr<Element> element) {
     m_elements.push_back(std::move(element));
 }
 
-float DisplayLayout::GetHeightFromDisplayPosition(
-    DisplayPosition pos, const Vector2 &displaySize) const {
+float GetHeightFromDisplayPosition(DisplayPosition pos,
```

Comment:
I would make this a static function since it is only used in this file to restrict its visibility.